sentry-ruby/spec/sentry/event_spec.rb

```diff
@@ -77,6 +77,85 @@
     end
   end
 
+  context 'rack context specified' do
+    require 'stringio'
+
+    before do
+      Sentry.init do |config|
+        config.dsn = DUMMY_DSN
+      end
+
+      Sentry.get_current_scope.set_rack_env(
+        'REQUEST_METHOD' => 'POST',
+        'QUERY_STRING' => 'biz=baz',
+        'HTTP_HOST' => 'localhost',
+        'SERVER_NAME' => 'localhost',
+        'SERVER_PORT' => '80',
+        'HTTP_X_FORWARDED_FOR' => '1.1.1.1, 2.2.2.2',
+        'REMOTE_ADDR' => '192.168.1.1',
+        'PATH_INFO' => '/lol',
+        'rack.url_scheme' => 'http',
+        'rack.input' => StringIO.new('foo=bar')
+      )
+    end
+
+    let(:event) do
+      options = described_class::Options.new(
+        level: 'warning',
+        tags: {
+          'foo' => 'bar'
+        },
+        extra: {
+          'my_custom_variable' => 'value'
+        },
+        server_name: 'foo.local'
+      )
+
+      Sentry::Event.new(
+        options: options,
+        configuration: Sentry.configuration,
+      )
+    end
+
+    context "without config.send_default_pii = true" do
```

Comment:
@HazAT can you review this and the below scenarios to see if I missed any field/information to ignore?